src/lib/paginator/paginator.scss

```diff
@@ -0,0 +1,65 @@
+.mat-paginator {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  min-height: 56px;
```

Comment:
The 56px is kinda magic and seems to be repeated. Move it into a variable? Same goes for some of the other hardcoded numbers.